coderd/httpapi/username.go

```diff
@@ -13,14 +14,18 @@ var (
 )
 
 // UsernameValid returns whether the input string is a valid username.
-func UsernameValid(str string) bool {
+func UsernameValid(str string) (bool, error) {
```

Comment:
Since `bool` and `error` both seem to convey the same meaning, should we just drop the `bool` return value?

Reply:
Good point ...